aspnetcore/blazor/call-web-api.md

```diff
@@ -28,9 +28,21 @@ See the following components in the *BlazorWebAssemblySample* sample app:
 
 ## Packages
 
-Reference the *experimental* [Microsoft.AspNetCore.Blazor.HttpClient](https://www.nuget.org/packages/Microsoft.AspNetCore.Blazor.HttpClient/) NuGet package in the project file. `Microsoft.AspNetCore.Blazor.HttpClient` is based on `HttpClient` and [System.Text.Json](https://www.nuget.org/packages/System.Text.Json/).
+Reference the [System.Net.Http.Json](https://www.nuget.org/packages/System.Net.Http.Json/) NuGet package in the project file.
 
-To use a stable API, use the [Microsoft.AspNet.WebApi.Client](https://www.nuget.org/packages/Microsoft.AspNet.WebApi.Client/) package, which uses [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/)/[Json.NET](https://www.newtonsoft.com/json/help/html/Introduction.htm). Using the stable API in `Microsoft.AspNet.WebApi.Client` doesn't provide the JSON helpers described in this topic, which are unique to the experimental `Microsoft.AspNetCore.Blazor.HttpClient` package.
+To use a stable API, use the [Microsoft.AspNet.WebApi.Client](https://www.nuget.org/packages/Microsoft.AspNet.WebApi.Client/) package, which uses [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/)/[Json.NET](https://www.newtonsoft.com/json/help/html/Introduction.htm). Using the stable API in `Microsoft.AspNet.WebApi.Client` doesn't provide the JSON helpers described in this topic, which are unique to the `System.Net.Http.Json` package.
```

Comment:
Should we really say this? By the time Blazor goes to GA, this API will also be GA.

@danroth27 thoughts?

Reply:
We can yank it at GA tho ... I think that was the original idea.

Reply:
If users are using preview bits, they already are using non-GA bits. So I think that is fine. But would like to leave it to @danroth27 to choose.

Reply:
Blazor Server is shipped and stable, so I guess it's reasonable that a Blazor Server user would prefer to use Microsoft.AspNet.WebApi.Client until System.Net.Http.Json has shipped.

Reply:
Although we orient readers for Blazor Server a bit in the text, we open this topic by saying ...

> The guidance in this topic only pertains to Blazor WebAssembly apps.

... this remark sort'a kind'a is angled to just the WASM case.

Reply:
Related question ... should we go ahead and make the title of this topic ...

> Call a web API from ASP.NET Core Blazor WebAssembly

Reply:
I agree with @guardrex and I also read this as a `WASM`-only doc.

Reply:
I guess having this doc be WebAssembly specific makes sense. Calling a web API from a Blazor Server app should be no different than calling an API from any ASP.NET Core app. So, if this doc is WebAssembly specific, I think we can drop any mention of the Microsoft.AspNet.WebApi.Client and just tell people to use System.Net.Http.Json, which is what we recommend.

Reply:
Broader point related to our offline discussion: Blazor is rather difficult to cleanly document the way that we feel compelled to cover it with minimal duplicated content. It has a bit of a split personality disorder:

* Blazor WASM-only content
* Blazor Server and WASM content
* Blazor Server-only content

... with Blazor Server being halfway (or perhaps more than halfway) covered in the ASP.NET Core docs because it's very much is an ASP.NET Core app with Razor components and SignalR.

An alternative to the whole scheme here (and devs have asked for this) is a **_completely separate full coverage for WASM and Server_**. That's all docs in two versions, one for WASM and one for Server ... even if content is duplicated ... and even if the content duplicates ASP.NET Core documentation. That would clear up a lot of reader confusion and angst ... *however* ... at a rather 💰 *high cost* 💰, of course.

Reply:
That's a much broader decision and it can't happen now. Let's scope this change down to Blazor WASM.

Reply:
Didn't want to appear rude above, @guardrex. I respect your opinion, but that decision is too big to discuss here.

Reply:
I'll make those updates now.